tests/src/test/scala/cats/tests/NestedTests.scala

```diff
@@ -18,7 +18,7 @@ class NestedTests extends CatsSuite {
 
   {
     // Invariant composition
-    implicit val instance = ListWrapper.invariant
+    implicit val instance = Nested.catsDataInvariantForNested(ListWrapper.invariant, ListWrapper.invariant)
```

Comment:
Is there a reason that we need to create these instances for `Nested` (here and below) explicitly?